pkgs/by-name/ll/llama-cpp/package.nix

```
@@ -146,7 +156,7 @@ effectiveStdenv.mkDerivation (finalAttrs: {
     license = licenses.mit;
     mainProgram = "llama-cpp-main";
     maintainers = with maintainers; [ dit7ya elohmeier ];
-    broken = (effectiveStdenv.isDarwin && effectiveStdenv.isx86_64) || lib.count lib.id [openclSupport blasSupport rocmSupport cudaSupport] == 0;
+    broken = (effectiveStdenv.isDarwin && effectiveStdenv.isx86_64) || lib.count lib.id [openclSupport blasSupport rocmSupport cudaSupport vulkanSupport] == 0;
```

This isn't broken on x86_64-darwin anymore 🎉 
```suggestion
    broken = (effectiveStdenv.isDarwin && effectiveStdenv.isx86_64) || lib.count lib.id [openclSupport blasSupport rocmSupport cudaSupport] == 0;
```

---

Wait, this check is just off. Upstream's version is `broken = (useMetalKit && !effectiveStdenv.isDarwin);` and that's correct.